site/content/en/docs/formats/synthia.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'SYNTHIA'
+linkTitle: 'SYNTHIA'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original SYNTHIA dataset is available
+[here](https://synthia-dataset.net).
+
+Supported annotation types:
+- `Mask`
+
+## Import SYNTHIA dataset
+
+A Datumaro project with a SYNTHIA source can be created in the following way:
+
+```bash
+datum create
+datum import --format synthia <path/to/dataset>
+```
+
+It is also possible to import the dataset using Python API:
+
+```python
+from datumaro.components.dataset import Dataset
+
+synthia_dataset = Dataset.import_from('<path/to/dataset>', 'synthia')
+```
+
+SYNTHIA dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+dataset/
+├── GT/
+│   └── LABELS/
+│       ├── Stereo_Left/
+│       │   ├── Omni_B/
+│       │   │   ├── 000000.png
+│       │   │   ├── 000001.png
+│       |   |   └── ...
+│       │   └── ...
+│       └── Stereo_Right/
+│           ├── Omni_B/
+│           │   ├── 000000.png
+│           │   └── 000001.png
+│           |   └── ...
+│           └── ...
+└── RGB/
+    ├── Stereo_Left/
+    │   ├── Omni_B/
+    │   │   ├── 000000.png
+    │   │   ├── 000001.png
+    |   |   └── ...
+    │   └── ...
+    └── Stereo_Right/
+        ├── Omni_B/
+        │   ├── 000000.png
+        │   └── 000001.png
+        |   └── ...
+        └── ...
+```
+
+- `RGB` folder containing standard RGB images used for training.
+- `GT/LABELS` folder containing containing PNG files (one per image).
+  Annotations are given in tree channels. The third channel contains
```

Comment:
```suggestion
  Annotations are given in three channels. The third channel contains
```

Also, "third" is a bit ambiguous when it comes to color images. It would be more clear if it said "red channel" or "blue channel".

Reply:
Fixed